CMakeLists.txt

```diff
@@ -42,6 +49,11 @@ set(QFIELD_INCLUDE_SUBDIR ${DEFAULT_INCLUDE_SUBDIR} CACHE STRING "Subdirectory w
 set(WITH_VCPKG OFF CACHE BOOL "Build with vcpkg packaging layout")
 set(USE_MAC_BUNDLING NOT ${WITH_VCPKG} CACHE BOOL "Use mac bundling")
 
+set(RELATIVE_PREFIX_PATH ${WITH_VCPKG} "Use a prefix path relative to the application itself rather than hard coding the path while compiling")
```

Comment:
```suggestion
set(RELATIVE_PREFIX_PATH ${WITH_VCPKG} CACHE BOOL "Use a prefix path relative to the application itself rather than hard coding the path while compiling")
```